Docker/api-mssql.Dockerfile

```diff
@@ -6,7 +6,7 @@
 #tag 6.0-alpine 
 FROM mcr.microsoft.com/dotnet/aspnet@sha256:201cedd60cb295b2ebea7184561a45c5c0ee337e37300ea0f25cff5a2c762538
 LABEL maintainer="Ed-Fi Alliance, LLC and Contributors <techsupport@ed-fi.org>"
-ENV VERSION="0.0.0-alpha.0.65"
+ARG VERSION=latest
```

Comment:
"latest" will not work in the URL on line 22. It works for npm, but not NuGet.

For the ODS/API, I wrote a script called `get-version.ps1` that uses the NuGet API and some PowerShell to find the correct version for download. You can see this working in [my pull request](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/724/files). In your browser, search for "get-versions.ps1" to quickly find all references and the script itself.

Reply:
Oh, but yes I see/remember now... up above in `on-prerelease.yml` the version is set correctly. This seems fine to me then.